pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -304,6 +306,12 @@ func (manager *Manager) UpdateClusterStatus(ctx context.Context, name string, st
 }
 
 func buildClusterConfig(cluster *clusterv1alpha2.PediaCluster) (*rest.Config, error) {
+	if len(cluster.Spec.Kubeconfig) != 0 {
+		return clientcmd.BuildConfigFromKubeconfigGetter("", func() (conf *clientcmdapi.Config, err error) {
```

Comment:
I think you can use `clientcmd.NewClientConfigFromBytes()` here

Reply:
make sense, updated.